api/src/main/java/eu/coatrack/api/Proxy.java

```diff
@@ -153,6 +157,30 @@ public void setPort(Integer port) {
         this.port = port;
     }
 
+    public boolean isForwardCookie() {
```

Comment:
This could be made redundant using Lombok.

Reply:
I would like to discuss about this on our Telco in the afternoon as I am not understanding what you mean with making it public as the isForwardCookie is already public

Reply:
So as we discussed, I applied the @Getter and @Setter to the booleans and removed the previous getters and setters.
So I consider this thread resolved.

Reply:
Regarding using the Lombok for the whole Proxy Object, I prefer to do it in a follow-up issue (that way we can optimize and clean the whole code at once)